modules/polyfills/test/fetch-node/fetch.node.spec.js

```diff
@@ -26,3 +27,46 @@ test('fetch polyfill (Node.js)#fetch() ignores url query params when loading fil
   }
   t.end();
 });
+
+test('fetch polyfill (Node.js)#fetch() able to handle "Accept-Encoding: gzip"', async t => {
+  if (!isBrowser) {
+    const headers = {
```

Comment:
Since our request implementation now always accept these we could just add them to the headers list inside the request call, so that the app doesn't need to.

Reply:
No need to specify, those TC's are just to make sure if it's defined specifically, it's working (as well as other 2).